### PR TITLE
use std lib function in pmmr for all_ones() and most_significant_pos()

### PR DESCRIPTION
**Pull request by RCasatta**
Use standard library function instead of manually calculating the value of ones and the most significant pos.

### CODE REVIEW

**Comment by yeastplume** [comment]
Thanks for submitting this! Though we have problems with travis CI, the test failures in /chain are genuine this time around. Could you have a look?

**Comment by RCasatta** [comment]
At the moment I can't reproduce the travis failure on my machine, both `chain` and `grin` dirs are passing the tests. I am using a slightly older version, I am trying again after upgrading `rustc`

**Comment by RCasatta** [comment]
It's very strange, sometimes I've got the error, sometimes I havent, need more time to investigate

File: core/src/core/pmmr.rs [hunk 1]
```diff
@@ -987,28 +987,12 @@ fn bintree_jump_left(num: u64) -> u64 {
...
-	}
-	let mut bit = 1;
-	while num >= bit {
-		if num & bit == 0 {
-			return false;
-		}
-		bit = bit << 1;
-	}
-	true
+	num.leading_zeros() + num.count_ones() == 64
```
Comment (hashmap) [inline core/src/core/pmmr.rs]: looks like `0` will produce `true`

Comment (RCasatta) [reply core/src/core/pmmr.rs]: you are very right

**Comment by hashmap** [comment]
LGTM

**Pull request merged**